bin/cluster

```diff
@@ -38,6 +38,21 @@ class Cluster(object):
             deployment_type = os.environ['OS_DEPLOYMENT_TYPE']
         return deployment_type
 
+
+    def get_product_type(self, args):
+        """
+        Get the product_type based on the environment variables and the
+        command line arguments
+        :param args: command line arguments provided by the user
+        :return: string representing the product type
+        """
+        product_type = 'openshift'
+        if args.product_type:
```

Comment:
You are checking for a product_type argument, but the argument is never actually created.  It should be added to the meta_parser similar to the way deployment_type is added: https://github.com/ashcrow/openshift-ansible/blob/ae-ansible-merge-services-configs/bin/cluster#L269-L271